<h1>Steven B Smith</h1>
<p><strong>Email:</strong> sbs@stevenbsmith.net<br />
<strong>Website:</strong> <a href="http://stevenbsmith.net">stevenbsmith.net</a><br />
<strong>Location:</strong> Philadelphia region  </p>
<hr />
<h2>INDUSTRY EXPERIENCE</h2>
<h3><strong>Labcorp, Baltimore, MD</strong></h3>
<p><strong>Senior Bioinformatics Scientist</strong> <em>(May 2023 - Present)</em><br />
- Managing a team of engineers and scientists, providing guidance on customer support software, automated testing systems, and design control software development.<br />
- Leading software development efforts for sequencing-based &amp; FDA-cleared oncology diagnostics software (Python, Linux, Perl, SQL).<br />
- Visualizing &amp; analyzing high throughput sequencing data using established and newly developed workflows (Python, R).  </p>
<h3><strong>Output Biosciences, New York, NY</strong></h3>
<p><strong>Computational Biology Lead</strong> <em>(June 2022 - April 2023)</em><br />
- Built vaginal and gut microbiome bioinformatics and computational infrastructure to extract value from clinical sample data (AWS, Python, Docker).<br />
- Designed, executed, and compiled statistical analysis reports communicated to leadership (Python).  </p>
<h3><strong>Kaleido Biosciences, Lexington, MA</strong></h3>
<p><strong>Principal Scientist, Computational Biology</strong> <em>(August 2021 - April 2022)</em><br />
- Discovered novel therapeutics that modulate the microbiome for treating immunological diseases using computational and statistical pipelines.<br />
- Led the design and build of a platform-wide analytical toolbox for biologists and chemists to query, integrate, analyze, visualize, and export experimental data (R, RShiny).<br />
- Analyzed 16S rRNA, shallow shotgun, and RNA sequencing data using Bioconductor &amp; DESeq/edgeR (R, Python, Linux, AWS).  </p>
<h3><strong>Teva Pharmaceuticals, West Chester, PA</strong></h3>
<p><strong>Principal Computational Biology Data Scientist</strong> <em>(July 2020 - August 2021)</em><br />
- Developed methods to prioritize, predict, and evaluate disease indication prioritization efforts using chromatin remodeling data from ENCODE and NCBI Databases.<br />
- Identified candidate clinical biomarkers using genomics, transcriptomics, and proteomics from patient data (R, RShiny).<br />
- Supported and consulted on various ongoing translational statistics studies (R, Python).  </p>
<h3><strong>Personal Genome Diagnostics, Baltimore, MD</strong></h3>
<p><strong>Computational Biology Scientist</strong> <em>(February 2020 - June 2020)</em><br />
- Developed innovative ctDNA NGS-based machine learning and computational methods to detect key genetic cancer signatures in liquid biopsies (Python, R, Linux).<br />
- Ensured product and customer requirements were met for several RUO assays.  </p>
<p><strong>Bioinformatics Scientist</strong> <em>(June 2017 - February 2020)</em><br />
- Designed both lab and in silico studies to evaluate performance of bioinformatics pipeline outputs.<br />
- Managed a team of software engineers to develop NGS bioinformatics pipelines.<br />
- Led the development of a Minimal Residual Disease (MRD) IUO monitoring assay using tissue and ctDNA plasma approaches.  </p>
<h3><strong>GlaxoSmithKline Pharmaceuticals, Upper Merion, PA</strong></h3>
<p><strong>Computational Biology Intern</strong> <em>(September 2010 - July 2012)</em><br />
- Identified common genomic signatures of host responses after exposure to respiratory viruses and bacteria using computational methods (R, Bioconductor, NCBI datasets).<br />
- Conducted Connectivity-MAP analysis to identify potentially repurposed compounds.<br />
- Analyzed metagenomic sequencing data in diabetic and obese mouse models.  </p>
<hr />
<h2>COMPUTATIONAL &amp; BIOINFORMATIC EXPERIENCE</h2>
<ul>
<li>Evaluated and prioritized Next Gen Seq gene expression, genomics, and proteomics data in Python and R, utilizing machine learning techniques (TensorFlow, Random Forest).  </li>
<li>Investigated host-microbiome interactions using RNA- and DNA-seq pipelines (Tophat/BWA, DESeq, edgeR).  </li>
<li>Developed functional algorithms, workflows, and databases using Docker, Perl, Python, Linux, SQL, and AWS.  </li>
</ul>
<hr />
<h2>LABORATORY EXPERIENCE</h2>
<ul>
<li>Optimized an RNA extraction method for sequencing small RNA on Illumina HiSeq 4000.  </li>
<li>Designed experiments for quantifying microRNA and microbial species using qPCR.  </li>
<li>Conducted cell and bacterial culture experiments.  </li>
</ul>
<hr />
<h2>EDUCATION</h2>
<h3><strong>University of Maryland, College of Computer, Mathematical, and Natural Sciences</strong></h3>
<p><strong>Doctor of Philosophy (PhD) in Biological Sciences</strong> <em>(Graduated May 2017)</em><br />
- <strong>Mentor:</strong> Jacques Ravel, Institute for Genome Sciences, Baltimore, MD<br />
- <strong>Thesis:</strong> Identification and Characterization of Regulatory miRNAs and mRNAs in the Longitudinal Human Host Response to Vaginal Microbiota.  </p>
<h3><strong>University of Pennsylvania, School of Engineering &amp; Applied Science</strong></h3>
<p><strong>Master of Science (M.S.) in Engineering</strong> <em>(Graduated August 2011)</em><br />
- <strong>Thesis:</strong> Identification of Common Biological Pathways and Drug Targets in Respiratory Viral Infection Using Host mRNA Expression Profiles.  </p>
<h3><strong>Drexel University, School of Biomedical Engineering, Science &amp; Health Systems</strong></h3>
<p><strong>Bachelor of Science (B.S.) in Engineering</strong> <em>(Graduated Cum Laude, June 2010)</em><br />
- <strong>Major:</strong> Biomedical Engineering  </p>
<hr />
<h2>PEER-REVIEWED PUBLICATIONS</h2>
<ul>
<li>Edwards VL*, Smith SB* (<em>contributed equally</em>), et al. (2019). <em>The cervicovaginal microbiota-host interaction modulates Chlamydia trachomatis infection.</em> mBio. <a href="https://doi.org/10.1128/mBio.01548-19">DOI:10.1128/mBio.01548-19</a>  </li>
<li>Delgado-Diaz, D.J., â€¦ Smith SB, et al. (2022). <em>Lactic acid from vaginal microbiota enhances cervicovaginal epithelial barrier integrity by promoting tight junction protein expression.</em> Microbiome. <a href="https://doi.org/10.1186/s40168-022-01337-5">DOI:10.1186/s40168-022-01337-5</a>  </li>
<li>Smith SB and Ravel, J. (2017). <em>The vaginal microbiota, host defense, and reproductive physiology.</em> J Physiol, 595: 451â€“463. <a href="https://doi.org/10.1113/JP271694">DOI:10.1113/JP271694</a>  </li>
<li>Smith SB, Magid-Slav M, Brown JR. (2013). <em>Host Response to Respiratory Bacterial Pathogens as Identified by Integrated Analysis of Human Gene Expression Data.</em> PLoS ONE 8(9): e75607.  </li>
<li>Smith SB, Dampier W, Tozeren A, Brown JR, Magid-Slav M. (2012). <em>Identification of Common Biological Pathways and Drug Targets Across Multiple Respiratory Viruses Based on Human Host Gene Expression Analysis.</em> PLoS ONE 7(3): e33174.  </li>
</ul>
<hr />
<p>This Markdown version keeps the structure of your resume intact while making it easy to format and read in a text-based environment. Let me know if you need any modifications! ðŸš€</p>